docs/usermanuals/en/data/pii-ml.md

```
@@ -0,0 +1,140 @@
+| `person` | medium | A name attached to other context can re-identify; alone it's weaker than a national ID. |
+| `organization` | low | Public-record entities; leakage is closer to "this person works at X" than "this is X's home address." |
+| `location` | low | Same logic — geographic strings are usually de-identification-resistant on their own. |
+
+These tiers sit **deliberately below** the regex `critical`/`high` floors (credit cards, national IDs). NER false-positive rates are materially higher than regex-anchored detection, so a "person" finding shouldn't gate a deployment the way a "credit_card" finding does.
+
+## Language selection
+
+Pass `--pii-ml-language` to point at a non-English NLP engine. Presidio raises a typed exception when no engine is registered for the requested language — surfaced loudly so you can install the matching spaCy model:
+
+```shell
+$ python -m spacy download xx_ent_wiki_sm     # multilingual, smaller
```

markdownlint-cli2 (0.22.1)

[warning] Dollar signs used before commands without showing output

(MD014, commands-show-output)